src/module/likes.js

```diff
@@ -0,0 +1,37 @@
+const urlLike = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${process.env.API_KEY}/likes/`;
+
+export const updateLikes = async (id, likes) => {
+  try {
+    const response = await fetch(urlLike);
+    const data = await response.json();
+    data.filter((item) => {
+      if (item.item_id === `item${id}`) {
+        likes.innerHTML = `${item.likes} likes`;
+      } else {
+        return '';
+      }
+      return '';
```

Comment:
- [ ] When a user clicks on the Like button of an item, the interaction should get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270), and the screen gets updated. Now screen is not updated. Kindly fix it.
![Screenshot from 2022-12-01 18-01-48](https://user-images.githubusercontent.com/65860678/205101834-68d6932b-aad0-4c73-a599-9e99fd458008.png)